packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2372,9 +2372,10 @@ export class ContainerRuntime
 			addBlobToSummary(summaryTree, idCompressorBlobName, idCompressorState);
 		}
 
-		if (this.remoteMessageProcessor.partialMessages.size > 0) {
-			const content = JSON.stringify([...this.remoteMessageProcessor.partialMessages]);
-			addBlobToSummary(summaryTree, chunksBlobName, content);
+		const partialMessages =
+			this.remoteMessageProcessor.getPartialMessageInfoForSummary(telemetryContext);
+		if (partialMessages !== undefined) {
+			addBlobToSummary(summaryTree, chunksBlobName, JSON.stringify(partialMessages));
```

Comment:
@kian-thompson - I wrote a bug! 😵

Introducing the variable, I dropped the array spread, so this is stringifying the Map, which doesn't do what you want.  Yikes!

And apparently this feature has no test coverage 😅

Reply:
I ran into the same thing when I initially developed this:
https://github.com/microsoft/FluidFramework/pull/10811/files#diff-a603c71ce072b3f8c60c6f239e50fb32923aef91d28d540914162b1dfa1a3a98